Document/0x07-Security-Testing-SDLC.md

```diff
@@ -86,25 +88,42 @@ The following diagram provides an overview of the way to perform test in the SDL
-* Penetration Testing: this is the "king" of security tests, the one that is the most famous and often performed. However, one must keep in mind that they happen late in the development lifecycle and that they cannot find every kind of flaw. They are too often constrained by available resources (time, money, expertise, ...), and as such should be complemented by other kind of tests. The current guide is about pentesting, and the reader will find a lot of useful information to conduct added-value tests and find even more vulnerabilities. Pentesting techniques include vulnerability scanning and fuzzing; however, pentests are much larger than these two examples. �Useful tools are listed in another section.
+* **Static Analysis**: by nature, static analysis is about analysing source code without running it. The goal of this artefact is twofold: make sure the Secure Coding rules the team has agreed on are correctly implemented when writing code, and finding vulnerabilities. Often, specialized software tools are used to automate this task, as hundreds and thousands of lines of code may need to be analysed. However, the drawback is that tools can only find what they have been told to look for and, today, are not as successful as human beings. This is the reason why sometimes Static Analysis is performed by humans (in addition to tools or not): it may take more time to humans, but they have a more creative way to detect vulnerabilities.�Examples of tools for Static Analysis are given in another section.
+* **Unit Tests**: unit tests make up the family of tests that are the closest to source code (e.g. they are focused on a single unit) as they are performed along with code. According to the methodology in use, they can be created either before code is developed or right after. Whatever the case, the end goal is to verify that procuced code is behaving as expected, but also that proper controls are put in place to prevent abuse cases (input filtering / validation, whitelisting, ...) and cannot be circumvented. Unit Tests are used to detect issues early in the development lifecycle to be able to fix them as soon and easily as possible. They are different from other forms of tests, like Integration / Verification / Validation tests, and may not be used to detect the same kind of issue. �Often, Unit Tests are aided with tools; a few of them are listed in another section.

```

Comment:
It seems sometimes are some unicode characters there that cannot be displayed, for example "�Often" and also in other areas of the PR. Can you please clean this?